pkg/cable/libreswan/libreswan.go

```diff
@@ -186,6 +187,31 @@ func extractSubnets(endpoint subv1.EndpointSpec) []string {
 	return subnets
 }
 
+func whack(args ...string) error {
+
+	var err error
+	for i := 0; i < 3; i++ {
+		cmd := exec.Command("/usr/libexec/ipsec/whack", args...)
+		cmd.Stdout = os.Stdout
+		cmd.Stderr = os.Stderr
+
+		klog.V(log.TRACE).Infof("Whacking with %v", args)
+
+		if err = cmd.Run(); err == nil {
+			break
+		}
+
+		klog.Errorf("error %v whacking with args: %v", err, args)
```

Comment:
```suggestion
		klog.V(log.DEBUG).Infof("Error whacking with args: %v - retrying: %v", args, err)
```

Reply:
Even if it's retriable, it's an error, it's not supposed to happen.

May be we should keep it as warning instead? (middle ground ;D )

Reply:
yeah warning sounds good - it's not an actual error until we're out of retries.